geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/TiledRasterRDD.scala

```diff
@@ -211,6 +211,53 @@ abstract class TiledRasterRDD[K: SpatialComponent: AvroRecordCodec: JsonFormat:
     withRDD(rdd.convert(CellType.fromName(newType)))
 
   protected def withRDD(result: RDD[(K, MultibandTile)]): TiledRasterRDD[_]
+
+  def singleTileLayerRDD: TileLayerRDD[K] = TileLayerRDD(
+    rdd.map({ case (k, v) => (k, v.band(0)) }),
+    rdd.metadata
+  )
```

Comment:
Yeah, that's the current state of the polygonal summary methods. Should I make it so that the operation is done per band and then do a final summary for all of the resulting values?

Also many other methods that are restricted to `Singleband` use this method in the backend.